src/app/component/WishlistItem/WishlistItem.component.js

```diff
@@ -111,22 +162,136 @@ export class WishlistItem extends PureComponent {
         };
     }
 
+    renderName() {
+        const { product: { name } } = this.props;
+
+        return (
+            <span>{ name }</span>
+        );
+    }
+
+    renderPrice(productPrice) {
+        return (
+            <div
+              block="WishlistItem"
+              elem="Price"
+            >
+                { productPrice() }
+            </div>
+        );
+    }
+
+    renderAttributes() {
+        const { attributes } = this.props;
+
+        return (
+            <div block="WishlistItem" elem="AttributeWrapper">
+                { attributes.map((attr) => (
+                    <span mix={ { block: 'ProductAttribute' } }>
+                        { attr }
+                    </span>
+                )) }
+            </div>
+        );
+    }
+
+    renderSelectCheckbox() {
+        const { product: { wishlist: { id } }, handleSelectIdChange, isEditingActive } = this.props;
+
+        return (
+            <div block="WishlistItem" elem="Select" mods={ { isEditingActive } }>
+                <Field
+                  type="checkbox"
+                  id={ `option-${ id }` }
+                  name={ `option-${ id }` }
+                  // eslint-disable-next-line react/jsx-no-bind
+                  onClick={ () => handleSelectIdChange(id) }
+                />
+            </div>
+        );
+    }
+
+    renderContentMobile({
+        content: { productPrice },
+        pictureBlock: { picture: renderPicture },
+        renderCardLinkWrapper
+    }) {
+        const { isEditingActive } = this.props;
+
+        return (
+            <>
+                <div block="WishlistItem" elem="SelectWrapper">
+                    { this.renderSelectCheckbox() }
+                    <div block="WishlistItem" elem="ContentWrapper" mods={ { isEditingActive } }>
+                        { renderCardLinkWrapper((
+                            <figure mix={ { block: 'ProductCard', elem: 'Figure' } }>
+                                { renderPicture({ block: 'WishlistItem', elem: 'Picture' }) }
+                            </figure>
+                        ), { block: 'WishlistItem', elem: 'ImageWrapper' }) }
+                        <div block="WishlistItem" elem="Content">
+                            { this.renderName() }
+                            { this.renderAttributes() }
+                            { this.renderQuantityField() }
+                            <div block="WishlistItem" elem="RowWrapper">
+                                { this.renderPrice(productPrice) }
+                                { this.renderAddToCartButton() }
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                { this.renderDescription() }
```

Comment:
This is extremely hard to read. Especially the `renderCardLinkWrapper` function call.